src/standalone/App.vue

```diff
@@ -1,20 +1,140 @@
 <template>
-  <div>
-    <router-view />
-  </div>
+  <v-app>
+    <!-- TODO: enable theme switching w/ dark and light props
+    -will need to swap techtonic icon too -->
+    <v-toolbar
+      app
+    >
+      <v-toolbar-side-icon
+        v-model="drawerLeft"
+        @click.stop="miniVariantLeft = !miniVariantLeft"
+      />
+      <v-toolbar-title class="headline">
+        <span>Techtonic</span>
+      </v-toolbar-title>
+      <v-spacer />
+      <v-btn
+        href="https://www.techtonic.com"
+        target="_blank"
+        flat
+      >
+        <v-img
+          :src="require('../assets/Techtonic_Black.png')"
+          contain
+        />
+      </v-btn>
+      <v-toolbar-side-icon
+        v-model="drawerRight"
+        @click.stop="drawerRight = !drawerRight"
+      />
+    </v-toolbar>
+    <v-navigation-drawer
+      v-model="drawerLeft"
+      persistent
+      :mini-variant="miniVariantLeft"
+      :clipped="clipped"
+      enable-resize-watcher
+      width="500"
+      fixed
+      app
+    >
+      <v-list>
+        <v-list-tile
+          v-for="(item, i) in itemsLeft"
+          :key="i"
+          value="true"
+        >
+          <v-list-tile-action>
+            <v-icon class="black--text">
+              {{ item.icon }}
+            </v-icon>
+          </v-list-tile-action>
+          <v-list-tile-content>
+            <v-list-tile-title v-text="item.title" />
+          </v-list-tile-content>
+        </v-list-tile>
+      </v-list>
+    </v-navigation-drawer>
+    <v-navigation-drawer
+      v-model="drawerRight"
+      persistent
+      :mini-variant="miniVariantRight"
+      :clipped="clipped"
+      enable-resize-watcher
+      fixed
+      right
+      app
+    >
+      <v-list>
+        <v-list-tile>
+          <v-list-tile-action>
+            <v-icon
+              large
+              class="black--text"
+            >
+              notifications
+            </v-icon>
+          </v-list-tile-action>
+        </v-list-tile>
+        <v-list-tile
+          v-for="(item, i) in itemsRight"
+          :key="i"
+          value="true"
+        >
+          <v-list-tile-action>
+            <v-icon class="black--text">
+              {{ item.icon }}
+            </v-icon>
+          </v-list-tile-action>
+          <v-list-tile-content>
+            <v-list-tile-title v-text="item.title" />
+          </v-list-tile-content>
+        </v-list-tile>
+      </v-list>
+    </v-navigation-drawer>
+    <v-content>
+      <!-- this displays the pages listed in router/routes.js -->
+      <router-view />
+    </v-content>
+  </v-app>
 </template>
 
 <script>
-export default {
-  data() {
-    return {};
+// import components for drawers here
+
+const App = {
```

Comment:
I prefer the default export rather than assigning a variable and exporting that but I can be convinced to go with this if there are good reasons

Reply:
I was imitating a tutorial from NetNinja and I like that it forces you to use the same name everywhere.  I'm not attached to either method so I'll change it to a default export.